docs/index.md

```diff
@@ -5,3 +5,20 @@ This website contains documentation for the scientific computing infrastructure
 ## About
 
 The sources of this documentation can be found on [GitHub](https://github.com/IKIM-Essen/ClusterDocs) and we encourage contribution.
+
+## Intro
+
+We believe it is important to know a thing or two about the underlying computer and network infrastructure to be effective. We note this will also limit your frustration level. We do not provide extensive documentation, but rather jumping off points and short best practice info on your setup and your procedures.
+
+  By configuring your environment correctly you can make your job easier, check out [Mamba/Conda](conda) for info on getting the right execution environment set up for your code, it might be desirable to run them inside a [container](apptainer) similar to the Docker system. Some users will benefit from using interactive [Jupyter Notebooks](jupyter).
```

Comment:
Slurm should probably be mentioned before anything else since the other guides refer to it.

Reply:
I think concepts need to be understood so order matters a bit less to me, but feel free to rephrase